perf_test/sparse/CMakeLists.txt

```diff
@@ -116,6 +116,13 @@ KOKKOSKERNELS_ADD_EXECUTABLE(
         SOURCES KokkosSparse_mdf.cpp
 )
 
+# Do not build this CRS sorting perf test by default.
```

Comment:
What's the issue here? Should this be simply guarded so it can be turned on with a CMake option?

Reply:
I disabled this just to avoid adding to the bulid time, since this is probably not something that will need to be benchmarked often. I can make it an option instead of a comment though.

Reply:
I think an option to turn it on would be better

Reply:
Now it's enabled by ``-DKokkosKernels_ENABLE_SORT_CRS_PERFTEST=ON`` (off by default).